Cargo.toml

```diff
@@ -0,0 +1,70 @@
+################################################################################
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# SPDX-License-Identifier: Apache-2.0
+################################################################################
+[package]
+name = "rust_tck"
+version = "0.1.0"
+edition = "2021"
+exclude = [
+    "tests/*",
+    "tools/*",
+    ".devcontainer/*",
+    ".vscode/*",
+    ".github/*",
+    ".gitignore",
+]
+include = ["test_agent/rust/src/main.rs",
```

Comment:
Not a typical usage I've seen, but will be resolved by suggestions given [here](https://github.com/eclipse-uprotocol/up-tck/pull/23/files#r1565803518).